amy/scripts/seed_training_requirements.py

```diff
@@ -22,13 +29,7 @@
 
 TRAINING_REQUIREMENTS: list[TrainingRequirementDef] = [
     {"name": "Training", "url_required": False, "event_required": True},
-    {"name": "DC Homework", "url_required": True, "event_required": False},
-    {"name": "SWC Homework", "url_required": True, "event_required": False},
-    {"name": "Discussion", "url_required": False, "event_required": False},
```

Comment:
"Discussion" is not being deprecated as it's being switched to "Welcome Session". The migration `0259_...` should probably only migrate the name "Discussion" <-> "Welcome Session", without changes the seeding manages.